Packs/SafeBreach/pack_metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-    "name": "SafeBreach - Breach and Attack Simulation platform (Deprecated)",
-    "description": "Deprecated. No available replacement.",
+    "name": "SafeBreach - Breach and Attack Simulation platform",
+    "description": "Breach and Attack Simulation platform",
     "support": "partner",
     "hidden": true,
     "videos": [
         "https://www.youtube.com/watch?v=Wb7q5Gbd2qo"
     ],
-    "currentVersion": "1.3.0",
+    "currentVersion": "1.3.1",
```

Comment:
its a major update, please make it 1.4.0 instead, and update the rn file name as well